Gemfile

```diff
@@ -52,7 +52,7 @@ group :development, :staging, :test do
 end
 
 group :staging, :production do
-  gem 'mysql', "~> 2.9.0"
+  gem 'mysql2', "~> 0.3"
```

Comment:
If I try to deploy with 0.4, it fails on this
```
LoadError: Please install the mysql2 adapter: `gem install activerecord-mysql2-ada
pter` (can't activate mysql2 (~> 0.3.10), already activated mysql2-0.4.2. Make sure all dependencies
 are added to Gemfile.)
```
which previously led me to try deploying with the pin at 0.3.10...